packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -835,6 +835,16 @@ export const prodParasKusamaCommon: EndpointOption[] = [
     ui: {
       logo: nodesBridgeHubBlackSVG
     }
+  },
+  {
+    homepage: 'https://krest.peaq.network/',
```

Comment:
This section is for common-good & system parahains.

You need to add it above.

Reply:
Thankyou done the changes , kindly review now

Reply:
@jacogr kindly let us know if there is anything pending from our-side